components/partial/code/CodeExamples.styled.ts

```diff
@@ -90,6 +93,15 @@ export const CodeBlock = styled.div`
         &symbol {
           color: ${CODE_BLOCK_ORANGE};
         }
+        &name {
```

Comment:
I added these because they were used by the XML syntax highlighter (see the Kotlin instructions). Kinda spitballed on the colors so open to using different ones.

Reply:
How about these for the colors?

>   string {
          color: ${CODE_BLOCK_YELLOW};
        }
     .
     .
        &name {
          color: ${CODE_BLOCK_PINK};
        }
        &tag {
          color: ${CODE_BLOCK_PINK};
        }
        &attr {
          color: ${CODE_BLOCK_GREEN};
        }

<img width="635" alt="Screen Shot 2020-06-04 at 4 19 42 PM" src="https://user-images.githubusercontent.com/44006085/83811422-61cb6b00-a67f-11ea-94fa-3ab4e37561c6.png">

Reply:
Nice! I'm going to leave this for a follow-up PR to give us time to get clarification from design and see what they had in mind.